e2e/manifests/devnet1.toml

```diff
@@ -5,9 +5,14 @@ anvil_chains = ["mock_l1", "mock_l2"]
 multi_omni_evms = true
 prometheus = true
 deploy_solve = true
+xcaller_enabled = true
 
 [node.validator01]
 [node.validator02]
 
 [node.fullnode01]
 mode="archive"
+
+[xcaller_chainid_pairs]
+mock_l1 = "mock_l2"
+mock_l2 = "mock_l1"
```

Comment:
Some thoughts on config.

- let's make it clear these are part of loadgen config
- we use chain names, not chain ids.
- do we have a reason for specifying which xcalls to to make xcalls from / to? if not, let's just do all of them. until we have a reason for more granular control

```toml
[loadgen.xcalls]
enabled = true
```

if we do have a need to specify, I think pairs are limitting. for ex, we could not loadtest xcalls on all streams by  configuring pairs (at least, not as written). What about just specifying source and destination chains with a regex

```toml
[loadgen.xcall]
from = "mock_l1|mock_l2"
to = "*"
```